keyboards/chonkerkeys/max/keymaps/default/config.c.template

```diff
@@ -12,6 +12,66 @@ const uint8_t layers[LAYER_COUNT] = {
     CH_ZOOM_MACOS
 };
 
+const uint16_t windows_configs[KEYCODE_COUNT][KEY_MACROS_MAX_COUNT] = {
```

Comment:
I think these configs are fixed across models/products and wouldn't need to be in config.c? The original design/intention is that anything inside `config.c` is generated from the json.

Reply:
Best to move them back inside config.c because of how QMK code is organized. They were originally in core.c but then I realized they lean more to something configurable per keyboard (much like macros) than core logic.